postgresql/defaults.yaml

```diff
@@ -5,10 +5,9 @@ Debian:
       - postgresql
     service:
       name: postgresql
+    cluster:
+      createcluster_bin: pg_createcluster
     config:
-      manage:
```

Comment:
Now i remember :+1: Without it in defaults.yaml you can decide if you want to manage the config files trough salt or not, this is what i needed during development.